google/cloud/pubsublite/internal/wire/partition_count_watching_publisher.py

```diff
@@ -56,8 +56,9 @@ async def __aenter__(self):
         return self
```

Comment:
It won't let me comment on unchanged lines. Can you add a test for _poll_partition_count_action failing the first time?

Reply:
Done.